fastify.js

```diff
@@ -471,7 +473,7 @@ function build (options) {
         _fastify._contentTypeParser,
         config,
         _fastify._errorHandler,
-        buildMiddie(_fastify._middlewares),
+        null,
```

Comment:
With this PR, is this parameter still necessary?

Reply:
Yes, here:
https://github.com/fastify/fastify/blob/c0c58d68c37aebe7e2b2306451a6a99e5d927fb1/fastify.js#L655-L668